_layouts/default.html

```diff
@@ -11,7 +11,6 @@
     <div class="site-container">
       {% include top-nav-bar.html %}
       <div class="content-container">
-        {% include hero.html %}
```

Comment:
😢

Reply:
@yuvenus was there a discussion that led to this being removed?

Reply:
unfortunately it was one of the things the HTML proofer was not happy about, nor did it have any immediate functionality, hence its removal :( however, if it's actually needed and the proofer needs to be wrangled, let me know and we can add it back in 👍

Reply:
@langiole https://github.com/citrineos/citrineos.github.io/pull/34 😄